pyprof/nvtx/nvmarker.py

```diff
@@ -68,14 +68,57 @@ def isfunc(mod, f):
     return ins.ismethod(attr) or ins.isfunction(attr) or ins.ismethoddescriptor(attr) or ins.isbuiltin(attr)
 
 
-def traceMarker(stack):
+def traceMarker():
```

Comment:
I would feel more comfortable with pushing this in if we had a unit test covering its behavior. 
Can we delay this merge until we have a test in place?

Reply:
I have manually run before and after and confirmed the resulting CSV files were the same other than minor timing changes

What test did you have in mind? Something specific for function stack creation?

Reply:
Ideally we make a unittest creating a hierarchy of function calls ie foo/bar and then verify that is what we see in the func_stack() output.
I am happy to take the action item if you wanna push til next week.

Reply:
Unit test is in.